packages/nviron/tests/index.test.ts

```
@@ -4,6 +4,14 @@ import { normalizeEnv } from "../src/utils/utils";
 import { validateEnv } from "../src/core/validator";
 import defineEnv from "../src/core/define-env";
 
+// Helper to strip ANSI color codes for easier assertion
+const stripAnsi = (str: string) =>
+  str.replace(
+    // eslint-disable-next-line no-control-regex
+    /[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-PRZcf-nqry=><]/g,
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)